main.go

```diff
@@ -15,18 +16,32 @@ import (
 	"github.com/rs/zerolog"
 )
 
+// Will be initialized at build time
+var version string
+
 func main() {
 	cfg := &gonfig.Config{}
 
-	var fsWatchInterval time.Duration
+	var fsWalkInterval time.Duration
 	var enableDebugLog bool
+	var kvImpl string
+	var versionFlag bool
```

Comment:
why not add all of this to Config?

Reply:
The version? Mm I'd keep that separate

`fsWalkInterval` it makes sense, I even tried but got an invalid type error with the duration field and I guess I was lazy to fix it

Reply:
fixed, was vscode trolling me.